prebid_cache_client/client.go

```diff
@@ -72,6 +75,10 @@ func (c *clientImpl) GetExtCacheData() (string, string) {
 	return c.externalCacheHost, c.externalCachePath
 }
 
+func (c *clientImpl) GetPutUrl() string {
```

Comment:
For this PR, we want to use the URL we can assemble from `getExtCacheData()`, we don't need a new method. When we return a URL to a client, we always want to return an external URL, not an PBS internal URLs, as they will be useless to the client if the internal differs from the external.